keps/sig-storage/1847-autoremove-statefulset-pvcs/kep.yaml

```diff
@@ -0,0 +1,17 @@
+title: Auto remove PVCs created by StatefulSet
```

Comment:
Please update this file with the missing fields as seen in the `kep.yaml` here: https://github.com/kubernetes/enhancements/blob/master/keps/NNNN-kep-template/kep.yaml

Thanks!

/Enhancement team